commit 37878307b83d6775ed56e5330418b4d346d93a5b
Author: Neko <neko@ayaka.moe>
Date:   Mon Jan 5 12:37:48 2026 +0800

    fix(docs): continue to fix Christmas time check

diff --git a/docs/.vitepress/composables/date.ts b/docs/.vitepress/composables/date.ts
index 2122bac1..ae871c50 100644
--- a/docs/.vitepress/composables/date.ts
+++ b/docs/.vitepress/composables/date.ts
@@ -13,5 +13,5 @@ export function isBetweenChristmasAndHalfOfJanuary(date: Date) {
   const christmas = new Date(year, 11, 20) // December 20
   const halfOfJanuary = new Date(year, 0, 10) // January 10 of the next year

-  return isAfter(date, christmas) && isBefore(date, halfOfJanuary)
+  return isAfter(date, christmas) || isBefore(date, halfOfJanuary)
 }
